react/features/overlay/middleware.ts

```diff
@@ -104,6 +109,8 @@ StateListenerRegistry.register(
             });
         }
 
+        dispatch(openDialog(PageReloadOverlay));
+
         if (NON_OVERLAY_ERRORS.indexOf(error.name) === -1 && typeof error.recoverable === 'undefined') {
```

Comment:
It's no longer necessary to set this fatal error, is it?

Reply:
Based on the logic that we have in AbstractPageReloadOverlay, I think it is.